code/modules/hydroponics/grown/melon.dm

```diff
@@ -14,6 +14,13 @@
 	mutatelist = list(/obj/item/seeds/watermelon/holy)
 	reagents_add = list("water" = 0.2, "vitamin" = 0.04, "nutriment" = 0.2)
 
+/obj/item/seeds/watermelon/suicide_act(mob/user)
+	user.visible_message("<span class='suicide'>[user] is swallowing [src]! It looks like [user.p_theyre()] trying to commit suicide!</span>")
+	user.gib()
```

Comment:
gibbing is probably a bad idea.

Reply:
C4 also gibs on suicide

Reply:
c4 is a bit rarer though

Reply:
oxygen tanks gib at max pressure, and they arent rare. how is this any worse?